dags/dbt_examples/copy_store_failures_bigquery.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from airflow.utils.dates import datetime
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from airflow.providers.google.cloud.transfers.bigquery_to_bigquery import (
+    BigQueryToBigQueryOperator)
+
+
+DBT_PROJ_DIR = os.getenv('DBT_PROJECT_DIR_BIGQUERY')
+DBT_PROFILE_DIR = os.getenv('DBT_PROFILE_DIR')
+DATASET = 'simple_bigquery_example_dag'
+AUDIT_PATH = f'{DATASET}_dbt_test__audit'
+FAIL_TABLE = 'accepted_values_forestfire_test_month__aug__mar__sep'
+
+with DAG('dbt.copy_store_failures_bigquery',
+         start_date=datetime(2021, 10, 8),
+         schedule_interval=None) as dag:
+    """
+    DAG to run dbt project and tests, then load the store_failures table into
+    a permament table so subsequent runs do not overwrite.
+
+    For the DAG to work, the following must exist:
+      - An Airflow Connection to GCP and BigQuery
+      - A BigQuery Dataset and Table created with forestfire data (can be
+          created by running the bigquery_examples.simple_bigquery_el DAG)
+      - A dbt profile with a connection to BigQuery in include/dbt/.dbt (.dbt
+          directory is .gitignored, this must be generated)
+    """
+
+    """
+    Run the dbt suite
+    """
+    dbt_run = BashOperator(
+        task_id='dbt_run',
+        bash_command=f"""
+        dbt run \
+        --profiles-dir {DBT_PROFILE_DIR} --project-dir {DBT_PROJ_DIR}
+        """
+    )
+
+    """
+    Run dbt test suite
```

Comment:
Only feedback here is that from the perspective of a new user (me), it's not clear reading this DAG what exactly the `dbt test` command is doing and what it's testing for. I don't know if that'd be obvious with someone with more dbt experience, but just wanted to call that out

Reply:
Gotcha. So the `dbt test` command runs the test suite which is specified under `include/dbt/[project]/models/[model]/schema.yml`. I can add this to the docstring.